File: computer-store-backend/models/computer.js
const mongoose = require('mongoose');

const computerSchema = mongoose.Schema({
    _id: mongoose.Schema.Types.ObjectId,
    name: {type: String, required: true},
    price: {type: Number, required: true},
    modelNumber: {type: String, required: true},
	imagePath: String,
	cpu: {type: String, required: true},
	rams: {type: Number, required: true},
	brand: {type: String, required: true},
});

module.exports = mongoose.model('Computer', computerSchema);

File: ComputerStore/ComputerStore/Model/Computer.swift
//
//  Computer.swift
//  ComputerStore
//
//  Created by Ali Rahal on 5/31/19.
//  Copyright © 2019 Ali Rahal. All rights reserved.
//

import Foundation

class Computer: NSObject, Codable {
    var _id: String?
    var name: String?
    var price: Float?
    var brand: String?
    var modelNumber: String?
    var imageURL: String?
    var cpu: String?
    var rams: Int?
}
